presto-main/src/test/java/com/facebook/presto/execution/TestTaskManagerConfig.java

```diff
@@ -32,14 +32,14 @@
     public void testDefaults()
     {
         assertRecordedDefaults(recordDefaults(TaskManagerConfig.class)
-                .setInitialSplitsPerNode(Runtime.getRuntime().availableProcessors() * 4)
+                .setInitialSplitsPerNode(Runtime.getRuntime().availableProcessors() * 2)
                 .setSplitConcurrencyAdjustmentInterval(new Duration(100, TimeUnit.MILLISECONDS))
                 .setStatusRefreshMaxWait(new Duration(1, TimeUnit.SECONDS))
                 .setInfoUpdateInterval(new Duration(200, TimeUnit.MILLISECONDS))
                 .setVerboseStats(false)
                 .setTaskCpuTimerEnabled(true)
                 .setMaxWorkerThreads(Runtime.getRuntime().availableProcessors() * 2)
-                .setMinDrivers(Runtime.getRuntime().availableProcessors() * 4 * 2)
+                .setMinDrivers(Runtime.getRuntime().availableProcessors() * 4)
```

Comment:
change this to * 2 * 2
to reflect that it is basically doing twice the number of max worker threads